src/docs/fund.yaml

```diff
@@ -0,0 +1,376 @@
+              schema:
+                $ref: "#/components/schemas/ValidationError"
+        500:
+          description: Internal Server Error
+  /api/fund/{id}:
+      get:
+        tags:
+          - Fund
+        summary: Get a fund
+        description: Get a fund by its id.
+        operationId: getFund
+        consumes:
+          - application/json
+        produces:
+          - application/json
+        parameters:
+          - name: id
+            in: path
+            required: true
+            schema:
+              type: string
+              example: 5e9f8f8f8f8f8f8f8f8f8f8
+        responses:
+          200:
+            description: The Fund object with the specified Id.
+            content:
+              application/json:
+                schema:
+                  $ref: "#/components/schemas/Fund"
+          422:
+            description: This status is returned when the request is missing required fields or has an invalid values.
+            content:
+              application/json:
+                schema:
+                  $ref: "#/components/schemas/ValidationError"
+          500:
+            description: Internal server error
+      delete:
+        tags:
+          - Fund
+        summary: Delete a fund
+        description: Delete a fund by its id.
+        operationId: deleteFund
+        consumes:
+          - application/json
+        produces:
+          - application/json
+        security:
+          - auth_cookie: []
+        parameters:
+          - name: id
+            in: path
+            required: true
+            schema:
+              type: string
+              example: 5e9f8f8f8f8f8f8f8f8f8f8
+        responses:
+          204:
+            description: The fund was deleted.
+            content:
+              application/json:
+                schema:
+                  type: object
+                  properties:
+                    message:
+                      type: string
+                      example: "Fund deleted"
+          403:
+            description: This status is returned when the user is not authorized to perform the operation.
+          422:
+            description: This status is returned when the request is missing required fields or has an invalid values.
+            content:
+              application/json:
+                schema:
+                  $ref: "#/components/schemas/ValidationError"
+          500:
+            description: Internal server error
+          401:
+            description: This status is returned if the user is not signed in.
+            content:
+              application/json:
+                schema:
+                  type: object
+                  properties:
+                    error:
+                      type: boolean
+                      example: true
+                    message:
+                      type: string
+                      example: "Invalid Token: No authorization token was found"
+components:
+  schema:
```

Comment:
Could you please change this to `schemas`. This is because it is mentioned in the whole file as schemas and not schema.

Reply:
it is written as **schema** in dev branch

Reply:
Ok, it should actually be written as schemas. You can correct it and push to the dev.